app/src/main/java/com/example/deejai/RoomSelection/Room.kt

```diff
@@ -2,9 +2,6 @@ package com.example.deejai.RoomSelection
 
 class Room(
     val name : String,
-    val coordinates : String,
+    val distance : String,
```

Comment:
Why are you changing this attribute? Wouldn't it make more sense to keep the room coordinate and use the getDistance method to calculate the distance?